app/templates/layouts/_base.html

```diff
@@ -114,6 +121,8 @@
 
     <!--[if gte IE 9]><!-->
     {% block scripts %}
+      {% set scripts_cdn = cdn_url_prefix~"/scripts/bundle.js" %}
```

Comment:
We should make sure that we are using the minified assets
eg `bundle.min.js`

Reply:
Changes for the above are now available to review.